cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+  });
+
+  describe('check modal with subtitle', () => {
+    it(`modal should contain subtitle`, () => {
+      cy.get('button#modal-trigger-5').click();
+      cy.get('#modal-5')
+        .find('.chi-modal__header')
+        .children()
+        .should('have.class', 'chi-modal__subtitle');
+      cy.get('#modal-5')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+  });
+
+  describe('check modal with inverse', () => {
+    it(`modal should contain inverse background`, () => {
+      cy.get('button#modal-trigger-6').click();
+      cy.get('#modal-6')
+        .parent()
+        .parent()
+        .should('have.class', '-inverse')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+  });
+
+  describe('check modal has no borders', () => {
+    it(`modal should not contain border for header`, () => {
+      cy.get('button#modal-trigger-7').click();
+      cy.get('#modal-7')
+        .find('.chi-modal__header')
+        .should('have.class', '-no-border')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+    it(`modal should not contain border for footer`, () => {
+      cy.get('button#modal-trigger-7').click();
+      cy.get('#modal-7')
+        .find('.chi-modal__footer')
+        .should('have.class', '-no-border')
+        .contains('Cancel')
+        .wait(550)
+        .click();
+    });
+  });
+  describe('check multi-step modal', () => {
+    it(`title of the modal should be aligned in the center of the header `, () => {
+      cy.get('button#modal-trigger-8').click();
+      cy.get('#modal-8')
+        .find('.chi-modal__title')
+        .should('have.class', '-centered');
+      cy.get('#modal-8')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+    it(`modal should have back button in the left side of the header`, () => {
+      cy.get('button#modal-trigger-8').click();
+      cy.get('#modal-8')
+        .find('.chi-modal__header')
+        .children()
+        .should('have.class', 'chi-modal__back');
+      cy.get('#modal-8')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+    it(`modal should have next button in the right side of the footer`, () => {
+      cy.get('button#modal-trigger-8').click();
+      cy.get('#modal-8')
+        .find('.chi-modal__footer')
+        .children()
+        .should('have.class', '-primary');
+      cy.get('#modal-8')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+  });
+
+  describe('check simple modal', () => {
+    it(`modal should not have header and footer`, () => {
+      cy.get('button#modal-trigger-9').click();
+      cy.get('#modal-9')
+        .should('not.have.class', 'chi-modal__header')
+        .and('not.have.class', 'chi-modal__footer')
+        .find('.-close')
+        .click();
+    });
+  });
+
+  describe('check portal modal', () => {
```

Comment:
The difference between the Base and Portal Modals is visual and it's checked by Backstop visual regression tests.
Let's remove the test case.

Reply:
Fixed in the latest commit